### PR TITLE
sheepChat app

### PR DESCRIPTION
**Pull request by creatic-pw**

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -2183,4 +2183,13 @@
   website: "https://englishextraapp.codeplex.com"
   repository: "https://github.com/englishextra/englishextra-app"
   icon: "englishextra-app.png"
+-
+  name: "sheepChat"
+  description: "Multi-chat for streamers"
+  website: "https://sheep.chat"
+  repository: ""
```
Comment (zeke) [inline _data/apps.yml]: Rather than specifying empty values, please remove `repository` and `keywords` entirely.

**Comment by creatic-pw** [comment]
@zeke done

**Comment by zeke** [comment]
One more thing, @creatic-pw. This needs to be rebased with master, as some other PRs have been merged recently. Thanks for taking the time to land this!

**Comment by kevinsawicki** [comment]
Merged to master as commit 10a9ffa1b0a75339306a8ed1a1c900f3df5aef58, thanks.

**Pull request closed without merging**